react/features/whiteboard/actionTypes.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Configure the whiteboard collaboration details.
+ * {{
+ *      type: SETUP_WHITEBOARD,
+ *      collabDetails
+ * }}
+ */
+export const SETUP_WHITEBOARD: string = 'SETUP_WHITEBOARD';
+
+/**
+ * Open the whiteboard collaboration session.
+ * {{
+ *      type: OPEN_WHITEBOARD,
+ *      participantId
```

Comment:
What does the participant ID mean here?

Reply:
That I forgot to clean up the jsdoc... :D